### ISSUE TITLE
vpm: doesn't work

### ISSUE THREAD

**Issue by Terisback** [vlang/v#7030]
**V version:** V 0.1.30 05eeb8a
**OS:** Ubuntu 20.04 LTS (VM)

**What did you do?**
`v install ui`

**What did you expect to see?**
installed ui module
 
**What did you see instead?**
```v
root@discord:~/dev# v install ui
cannot compile `/root/v/cmd/tools/vpm.v`: 
/root/v/thirdparty/cJSON/cJSON.o not found, building it in /root/.vmodules/cache/f2/f20fac5b113e1b4de6b14587bf1bfba6.o ...
failed thirdparty object build cmd:
/root/v/thirdparty/tcc/tcc.exe   -fPIC -I "/root/v/thirdparty/cJSON" -o "/root/.vmodules/cache/f2/f20fac5b113e1b4de6b14587bf1bfba6.o" -c "/root/v/thirdparty/cJSON/cJSON.c" 
builder error: In file included from /root/v/thirdparty/cJSON/cJSON.c:40:
In file included from /usr/include/string.h:33:
/usr/include/string.h:33: error: include file 'stddef.h' not found


root@discord:~/dev# 
```

**Comment by Terisback** [comment]
ui module has already been installed, but I wanted to update it

**Comment by spytheman** [comment]
temporary workaround:
`pushd . ; cd /root/v ;./v install ui; popd`

**Comment by spytheman** [comment]
Fixed in 7511d6e
